src/react-apps/applications/shared/__tests__/__snapshots__/LeftDrawerMenu.test.tsx.snap

```diff
@@ -489,7 +489,7 @@ exports[`LeftDrawerMenu.tsx render left drawer menu when mockmenutype is create
               </svg>
             </div>
             <div
-              className="MuiListItemText-root-85 LeftDrawerMenu-activeMenu-10"
+              className="MuiListItemText-root-85"
```

Comment:
Does this solution work? 

"You may have noticed that the class names generated by our styling solution are non-deterministic, so you can't rely on them to stay the same. The following CSS won't work:

.MuiAppBar-root-12 {
  opacity: 0.6
}
Instead, you have to use the classes property of a component to override them.".

Check out [material-ui introduction](https://material-ui.com/customization/css-in-js/#class-names) for more information. I do not think there is a guarantee that the classnames are consistent. 
@matsgm do you know anything more on this?

Reply:
@lorang92 I don't know if this is a problem with snapshot testing. Could be.

Reply:
Ok, then i will update and push the changes

Reply:
@lorang92 I tried setting it up in classes like
            classes={{
              paper: classNames(classes.drawer, classes.paper, {
                [classes.drawerOpen]: this.state.open,
                [classes.drawerClose]: !this.state.open,
              }),
            }}
But still i get different numbers on new build. But the snapshot tests doesn't fail because of this.